django/curator/management/commands/curator_cluster_tags.py

```diff
@@ -0,0 +1,59 @@
+import logging
+
+from django.core.management.base import BaseCommand
+
+from curator.tag_deduplication import TagClusterer
+
+
+class Command(BaseCommand):
+    help = "Cluster Tags using dedupe"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--run",
+            "-r",
+            help="Run the clustering model",
+            action="store_true",
+            default=False,
+        )
+
+        parser.add_argument(
+            "--label",
+            "-l",
+            help="Label the data via the console",
+            action="store_true",
+            default=False,
+        )
+
+        parser.add_argument(
+            "--save",
+            "-s",
+            help="Save the clustering results to the database",
+            action="store_true",
+            default=False,
+        )
+
+        parser.add_argument(
+            "--threshold",
+            "-t",
+            help="Threshold to use for clustering tags",
+            action="store_true",
+            default=0.5,
+        )
+
+    def handle(self, *args, **options):
+        tag_gazetteering = TagClusterer(clustering_threshold=options["threshold"])
```

Comment:
add heredocs to this method describing how to use this management command to bootstrap tag clusters properly and then how to transition to `curator_gazetteer_tags`.

naming a TagClusterer instance `tag_gazetteering` seems copypasta wrong.